plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -255,15 +259,18 @@ function auto_sizes_get_layout_width( string $alignment ): string {
  * @return string[] The filtered context keys used by the block type.
  */
 function auto_sizes_filter_uses_context( array $uses_context, WP_Block_Type $block_type ): array {
-	// The list of blocks that can consume outer layout context.
-	$consumer_blocks = array(
-		'core/cover',
-		'core/image',
+	// Define block-specific context usage.
+	$block_specific_context = array(
+		'core/cover'   => array( 'max_alignment' ),
+		'core/image'   => array( 'max_alignment', 'container_relative_width' ),
+		'core/group'   => array( 'max_alignment' ),
+		'core/columns' => array( 'max_alignment', 'container_relative_width' ),
+		'core/column'  => array( 'max_alignment', 'column_count' ),
```

Comment:
Do blocks still pass through parent context even when it's not allowlisted for `uses_context` here?

I'm asking since `core/group` and `core/column` don't use `container_relative_width`, but they'll still pass any value for `container_relative_width` down to their child blocks?

Reply:
Yes, `core/group` and `core/column` do not explicitly include `container_relative_width` in their `uses_context` array, but they still pass it down to child blocks since context is inherited by default.  

Do you think we should explicitly pass the `container_relative_width` context to the group block for added safety?

Reply:
Thanks for clarifying. No, I think the way you have it is good as is then.